pkg/agent/util/net.go

```diff
@@ -415,3 +415,17 @@ func GenerateRandomMAC() net.HardwareAddr {
 	buf[0] |= 2
 	return buf
 }
+
+func GetGlobalIPNetsByName(link *net.Interface) ([]*net.IPNet, error) {
```

Comment:
The function doesn't get IPs "by name". And I don't quite get what "global" means here. If it means "scope global", it doesn't do any check.

Reply:
Has changed to GetIPNetsByLink.